models/demos/mamba/tt/model_config.py

```diff
@@ -0,0 +1,182 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
```

Comment:
We can probably delete this since it is not used and is just copied from elsewhere.